webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -1,47 +1,120 @@
+import ChangesRequestedIcon from '../../images/icons/changes_requested';
 import {getLabelFontColor} from '../../utils/styles';
 
-function GithubItems(props) {
+const notificationReasons = {
+    assign:	'You were assigned to the issue',
+    author:	'You created the thread.',
+    comment:	'You commented on the thread.',
+    invitation:	'You accepted an invitation to contribute to the repository.',
+    manual:	'You subscribed to the thread (via an issue or pull request).',
+    mention:	'You were specifically @mentioned in the content.',
+    review_requested:	'You were requested to review a pull request.',
+    security_alert: 'GitHub discovered a security vulnerability in your repository.',
+    state_change: 'You changed the thread state.',
+    subscribed:	'You are watching the repository.',
+    team_mention:	'You were on a team that was mentioned.',
+};
+
+interface Label {
+    id: number;
+    name: string;
+    color: CSS.Properties;
+}
+
+interface User {
+    login: string;
+}
+
+interface Review {
+    state: string;
+    user: User;
+}
+
+interface Item {
+    url: string;
+    number: number;
+
+    id: number;
+    title: string;
+    created_at: string;
+    html_url: string;
+    repository_url?: string;
+    user: User;
+    owner?: User;
+    milestone?: {
+        title: string;
+    }
+    repository?: {
+        full_name: string;
+    }
+    labels?: Label[];
+
+    // PRs
+    status?: string;
+    mergeable?: boolean;
+    requestedReviewers?: string[];
+    reviews?: Review[];
+
+    // Assignments
+    pullRequest?: undefined;
+
+    // Notifications
+    subject?: {
+        title: string;
+    }
+    reason?: keyof typeof notificationReasons;
+}
+
+interface GithubItemsProps {
+    items: Item[];
+    theme: Theme;
+}
+
+function GithubItems(props: GithubItemsProps) {
     const style = getStyle(props.theme);
 
     return props.items.length > 0 ? props.items.map((item) => {
-        const repoName = item.repository_url ? item.repository_url.replace(/.+\/repos\//, '') : item.repository.full_name;
-
-        let userName = null;
+        let repoName = '';
+        if (item.repository_url) {
+            repoName = item.repository_url.replace(/.+\/repos\//, '');
+        } else if (item.repository?.full_name) {
+            repoName = item.repository?.full_name;
+        }
 
+        let userName = '';
         if (item.user) {
             userName = item.user.login;
         } else if (item.owner) {
             userName = item.owner.login;
         }
 
-        let title = item.title ? item.title : item.subject.title;
-        let number = null;
-
+        let number:JSX.Element | null = null;
         if (item.number) {
-            const iconProps = {
+            const iconProps: IconProps = {
                 size: 'small',
                 verticalAlign: 'text-bottom',
             };
 
             let icon;
-            if (item.pull_request) {
+            if (item.pullRequest) {
```

Comment:
Was this broken before?

Reply:
I think the JSON parser accepts both fields names. So no, it was not broken.